src/Tests/Microsoft.NET.TestFramework/TestAsset.cs

```diff
@@ -21,6 +21,9 @@ public class TestAsset : TestDirectory
 
         public string TestRoot => Path;
 
+        /// <summary>
+        /// The hashed test name (so file paths do not become too long) of a particular test. This will be the root of the test folder. 
```

Comment:
The hashing happens in `TestAssetsManager`, so I would probably note that here.  Otherwise it might be confusing to see this comment but not the code that does the hashing in `TestAsset`.

Reply:
Gotcha!